p2p_wallet/Info.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>DEFAULT_FF</key>
```

Comment:
Do we need declare all future feature in env file and then in info.plist? Or is it ok since new features are disabled by default.